package.json

```diff
@@ -7,6 +7,10 @@
     "node": "10.18.1",
     "npm": "5.6.0"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dvsa/cvs-tsk-update-store.git"
+  },
   "scripts": {
     "start": "npm run kill:offline && npm run start:offline && serverless invoke local -f main --path tests/resources/queue-event.json",
```

Comment:
We seem to be using the old template? Why are we not using the one Rob created?
It is not using webpack to bundle the code which has tree shaking, etc..

I think it is missing a few things DVSA require for us to start such the dvsa/eslint plugin

Reply:
@SanjeetBJSS is aware of this. Basically, it was a choice between:
* Using the old template and getting the work done
* Using the new template and not getting the work done

So he picked the first approach :D the new template at the start of this project was a constantly evolving piece of software, and Rob Waring also agrees this was the right choice

Note Tanio has also edited the package.json on my branch, some changes are his.

Reply:
Hi both, this was a 6 week activity and the template had not been signed off (and isn't). Given we are taking a release cut today and testing is complete, I do not want to change this now. I accept the technical debt incurred and I will pop a ticket in the backlog to address it. Hopefully  we can do this when we plug the activities data in. @sdh100shaun  @bristol-mike-jones , please shout if you have a different opinion.